module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,12 @@
+#include "sort.hpp"
+
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& strings) {
+    std::deque<std::string> sorted_strings{};
```

Comment:
Declare it later, and use {it.begin(), it.end()} to insert whole container during initialization.